From 258fa50cde0ac2e59e4726e05183e4999ff9213d Mon Sep 17 00:00:00 2001
From: Werner Lemberg <wl@gnu.org>
Date: Thu, 13 Feb 2020 16:20:47 +0100
Subject: [PATCH] Avoid empty locale while building LilyPond.

This is necessary because gettext versions >= 0.21 derive language
information from other system settings if `LANG` (or `LANGUAGE`) is not set.
---
 stepmake/stepmake/generic-vars.make | 2 +-
 stepmake/stepmake/texinfo-vars.make | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/stepmake/stepmake/generic-vars.make b/stepmake/stepmake/generic-vars.make
index 3c482bb415..59c1107a48 100644
--- stepmake/stepmake/generic-vars.make
+++ stepmake/stepmake/generic-vars.make
@@ -60,7 +60,7 @@ endif
 
 
 # no locale settings in the build process.
-LANG=
+LANG=C
 export LANG
 
 
diff --git a/stepmake/stepmake/texinfo-vars.make b/stepmake/stepmake/texinfo-vars.make
index 83709e32ef..92abdea855 100644
--- stepmake/stepmake/texinfo-vars.make
+++ stepmake/stepmake/texinfo-vars.make
@@ -32,7 +32,7 @@ DO_TEXI_DEP = ( echo ./$@: $(call scan-texi,$<) > $(basename $@).dep ) &&
 TEXINFO_PAPERSIZE_OPTION= $(if $(findstring $(PAPERSIZE),a4),,-t @afourpaper)
 
 MAKEINFO_FLAGS += --enable-encoding --error-limit=0 $(DOCUMENTATION_INCLUDES)
-MAKEINFO = LANG= $(MAKEINFO_PROGRAM) $(MAKEINFO_FLAGS)
+MAKEINFO = LANG=C $(MAKEINFO_PROGRAM) $(MAKEINFO_FLAGS)
 
 # texi2html xref map files
 XREF_MAPS_DIR = $(top-build-dir)/$(outdir)/xref-maps
-- 
2.16.4

